### DISCUSSION TITLE
SIF, Dynamic Weighting, Traffic

### DISCUSSION THREAD

**Discussion by schierkolk** [valhalla/valhalla#3708] [Q&A]
I am interested in slight altering the edge weight +/- a percentage of the actual cost for each solution. Is this best achieved using a SIF or something else be better ?

Chris

**Comment by kevinkreiser** [comment]
we currently do not support changing the cost of individual edges without making use of existing options and attribution on the edges in question. one option would indeed be to roll your own real time traffic corollary data set which would indeed allow you to vary the cost by modifying the speed of a given edge (not the cost directly but the speed which is a large component of the cost).

as you noticed in that other discussion i owe an explanation on traffic architecture and openlr matching and all thing things. i have a doc going but need some time to put it into place as markdown + issues for future contributions. please excuse the wait on this im stretched pretty thin at the moment
